app-sso/getting-started/provision-auth-server.hbs.md

```diff
@@ -24,209 +25,75 @@ tanzu package installed list -A | grep "sso.apps.tanzu.vmware.com"
-  namespace: default
-spec:
-  secretTemplate:
-    type: Opaque
-    stringData:
-      key.pem: $(privateKey)
-      pub.pem: $(publicKey)
+  name: my-unsafe-test-login-example
+EOF
 ```
 
-You can wait for the `AuthServer` to become ready with:
-
-```shell
-kubectl wait --for=condition=Ready authserver my-authserver-example
-```
+## Verify that your AuthServer is running
 
-Alternatively, you can inspect your `AuthServer` like any other resource:
+You can now see the service offering with:
 
 ```shell
-kubectl get authservers.sso.apps.tanzu.vmware.com --all-namespaces
+tanzu services classes list
 ```
 
-and you should see:
+You should see
 
 ```shell
-NAMESPACE NAME                  REPLICAS ISSUER URI                                         CLIENTS STATUS
-default   my-authserver-example 1        http://my-authserver-example.default.<your domain> 0       Ready
+NAME                          DESCRIPTION
+my-unsafe-test-login-example  Login by AppSSO - user:password - UNSAFE FOR PRODUCTION!
 ```
 
-As you can see your `AuthServer` gets an issuer URI templated. This is its entrypoint. You can find an `AuthServer`'s
-issuer URI in its status:
+__Note:__ As you can see this Login offering is *not* safe for production as it hard codes a user and password, but
+it is the quickest way to get started.
+
+You can wait for the `ClusterUnsafeTestLogin` to become ready with:
 
 ```shell
-kubectl get authservers.sso.apps.tanzu.vmware.com my-authserver-example -o jsonpath='{.status.issuerURI}'
+kubectl wait --for=condition=Ready clusterUnsafeTestLogin my-unsafe-test-login-example
 ```
 
-Open your `AuthServer`'s issuer URI in your browser. You should see a login page. Log in using username = `user` and
-password = `password`.
-
-You can review the standard OpenID information of your `AuthServer` by visiting
-`http://my-authserver-example.default.<your domain>/.well-known/openid-configuration`.
-
-> **Important** If the issuer URIs domain is not yours, the AppSSO package installation must be updated.
-For more information, see [Install Application Single Sign-On](../how-to-guides/platform-operators/installation.md).
-
-## The AuthServer spec in detail
+Alternatively, you can inspect your `ClusterUnsafeTestLogin` like any other resource:
 
-Here is a detailed explanation of the `AuthServer` you have applied in the above section. This is intended to give you
-an overview of the different configuration values that were passed in. It is not intended to describe all the
-ins-and-outs, but there are links to related docs in each section.
-
-Feel free to skip ahead.
-
-### Metadata
-
-```yaml
-metadata:
-  labels:
-    name: my-first-auth-server
-    env: tutorial
-  annotations:
-    sso.apps.tanzu.vmware.com/allow-client-namespaces: "default"
-    sso.apps.tanzu.vmware.com/allow-unsafe-issuer-uri: ""
-    sso.apps.tanzu.vmware.com/allow-unsafe-identity-provider: ""
+```shell
+kubectl get clusterUnsafeTestLogin.sso.apps.tanzu.vmware.com --all-namespaces
```

Comment:
**[suggestion]:** Conventionally, full resource names are all lower-case. You can consider acquainting the user with `cutl` as well.

Reply:
I can definitely lower case! I think @candicenonsense has an opinion on the `cutl` abbreviation if I remember correctly.

Reply:
I do not like the abbreviation.